client/lib/post-normalizer/rule-decode-entities.js

```diff
@@ -20,6 +20,9 @@ export default function decodeEntities( post, fields = DEFAULT_FIELDS ) {
 		}
 	} );
 
+	// Sometimes titles are double-encoded, so run again to be sure
+	post.title = decode( post.title );
```

Comment:
do we ever set the post title with `__dangerouslySetHtmlOrWhateverItsCalled`? If we do, this would open us up to an encoding attack...

Reply:
I've had an explore and we use the `dangerouslyDoScaryThings` on post excerpts and content, but never post titles as far as I can see.

Is there a better way around this than the double decode? An OSS citizen suggested doing the `dangerouslySetInnerHTML` on post titles instead: https://github.com/Automattic/wp-calypso/pull/37842/files

Reply:
I think it would be safe to use `dangerouslySetInnerHTML` because I think title is filtered before being saved into the database. I tried some random inputs but it didn't work. It looked like data is being filtered.

Reply:
Another way would be creating own unescape function but it would be limited to some special characters.

Reply:
https://stackoverflow.com/a/46851765
This is one of possible way to do this.